e2e/404.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { test, expect, type Page } from '@playwright/test';
+import words from '../client/i18n/locales/english/motivation.json';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/404');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe('404 Page Tests', () => {
+  test('should display the correct title', async () => {
```

Comment:
```suggestion
  test('should display the correct title', async ({ page }) => {
```

We can get `page` directly from the function args.